local/http/http.go

```diff
@@ -66,8 +79,22 @@ func (s *Server) Start(errChan chan error) (int, error) {
 	}
 	s.serverPort = strconv.Itoa(port)
 
+	gzipWrapper, err := gziphandler.GzipHandlerWithOpts(gziphandler.ContentTypes(gzipContentTypes))
+
+	if err != nil {
+		return port, err
```

Comment:
You need to wrap `err` with `errors.WithStack(err)` so that we can get stack traces in debug mode